src/clusterfuzz/_internal/bot/tasks/utasks/corpus_pruning_task.py

```diff
@@ -946,5 +950,13 @@ def utask_preprocess(fuzzer_name, job_type, uworker_env):
       corpus_pruning_task_input=corpus_pruning_task_input)
 
 
+_HANDLED_ERRORS = [
+    uworker_msg_pb2.ErrorType.CORPUS_PRUNING_FUZZER_SETUP_FAILED,
+]
+
+
```

Comment:
Could you write a crazy test that uses task_types to get every task module and then checks that all of its handled errors are in uworker_handle_errors dictionary?
Can come in another PR.

Reply:
Sure! I'll prepare that in a separate PR today or tomorrow morning.